src/AboutDialog.cpp

```diff
@@ -1129,7 +1134,29 @@ wxT("OUT OF THE USE OR INABILITY TO USE THE PROGRAM (INCLUDING BUT NOT LIMITED\n
 wxT("TO LOSS OF DATA OR DATA BEING RENDERED INACCURATE OR LOSSES SUSTAINED BY\n")
 wxT("YOU OR THIRD PARTIES OR A FAILURE OF THE PROGRAM TO OPERATE WITH ANY OTHER\n")
 wxT("PROGRAMS), EVEN IF SUCH HOLDER OR OTHER PARTY HAS BEEN ADVISED OF THE\n")
-wxT("POSSIBILITY OF SUCH DAMAGES.\n"));
+wxT("POSSIBILITY OF SUCH DAMAGES.\n");
+
+#if defined(HAS_PRIVACY_POLICY)
+   /* i18n-hint: For "About Audacity..." Title for Pivacy Policy section */
+   const auto privacyPolicyTitle = XC("PRIVACY POLICY", "about dialog");
+
+   const auto privacyPolicyText = XO(
+      "App update checking and error reporting require network access. "
+      "These features are optional. See our [[https://www.audacityteam.org/about/desktop-privacy-notice/|Privacy Policy]] "
+      "for more information.");
+
+   const wxString privacyPolicyHTML = wxString(
+      wxT("<center>") + 
+      privacyPolicyTitle.Translation() + 
+      wxT("</center><p>") +
+      privacyPolicyText.Translation() + 
+      wxT("</p><br/><br/>")
+   );
+
+   wxString PageText = FormatHtmlText(privacyPolicyHTML + GPL_TEXT);
```

Comment:
Privacy policy before GPL because we want the screen readers to say it first and otherwise make it prominent.